### PR TITLE
add metaphase blog post

### PR DESCRIPTION
**Pull request by jalehman**

### CODE REVIEW

**Comment by matildepark** [comment]
Image is broken.

File: content/blog/metaphase.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+So now, before we could launch a stable stream, we had some things to do first:
+**We wanted to provide assurance that the userspace suite is relatively stable.** All our remaining stores (invite-store, metadata-store, and contact-store) needed to be modernised to use the same libraries and utilities inside Arvo as newer userspace applications.
+**We wanted all our applications to be using graph-store.** If we were going to migrate the entire network to a new userspace model, we shouldn’t undermine the new release cycle process and make it more difficult to test and deploy to meet higher user expectations. We needed to  migrate everything first.
+**We wanted to release the notification-store**. After the October 2020 groups release, Landscape simply wouldn’t be complete without it.
+Finally, **we wanted all of Landscape to use the [Indigo component library](https://github.com/urbit/indigo-react).** Landscape shipped with each application using its own build with handwritten, bespoke DOM components. Once we moved to a single-page application architecture in June 2020, we wanted the same guaranteed consistency for both back-end and front-end code.
```
Comment (matildepark) [inline content/blog/metaphase.md]: This should be a list with bullet points, with a line break after L49 here.

**Review by matildepark — APPROVED** [review]

**Comment by jalehman** [comment]
@matildepark image is uploaded, list is bulleted. It looks good to me—take a final look, and then I'll make it go live.

**Comment by matildepark** [comment]
I did take a final look. That's why I stamped it.

**Comment by jalehman** [comment]
so fast

**Pull request merged**